tests/conftest.py

```diff
@@ -6,3 +6,8 @@
 @pytest.fixture(scope="module")
 def data_path():
     return Path(__file__).parent / "data"
+
+
+@pytest.fixture(scope="module")
+def pccs_url():
+    return "https://pccs.mse.cosmian.com"
```

Comment:
maybe use the PCCS of dev/staging instead for tests?

Reply:
Done in 5e7fee0.